manifests/t/the-sz/Bear/latest/the-sz.Bear.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Bear
+PackageVersion: latest

```

Comment:
```suggestion
PackageVersion: 1.60
```

Reply:
I used "PackageVersion: latest" on purpose, since I don't have version specific download links. They always point to the latest version.